src/net_processing.cpp

```diff
@@ -3900,6 +3900,8 @@ bool PeerLogicValidation::SendMessages(CNode* pto)
             connman->PushMessage(pto, msgMaker.Make(NetMsgType::INV, vInv));
 
         // Detect whether we're stalling
+        const auto current_time = GetTime<std::chrono::microseconds>();
+        // nNow is the current system time (not mockable) and should be removed eventually
```

Comment:
probably want to directly mention why nNow is mockable here

Reply:
It is *not* mockable

Added a comment